.gitlab-ci.yml

```diff
@@ -61,17 +61,12 @@ variables:
 # --------------------------------------
 
 format:
-  stage: formatter
+  stage: format-check
   image: $CI_REGISTRY/r3/docker/julia-custom
   script:
     - julia --project=@. -e 'using JuliaFormatter; format(".", verbose=true);'
     - git status
-    - git remote set-url origin "https://$CI_USER_NAME:$GITHUB_ACCESS_TOKEN@github.com/$GITHUB_REPOSITORY"
```

Comment:
why remove this auto-formatting?

Reply:
It is cool but for whatever reason broken in this form and I can't fix it, so I just axed it.

In the long term I'd kinda like to push everyone to always think about formatting himself and making sure that it looks good before typing `push`. :D

Reply:
yeah, but that won't happen - I want to keep the auto-formatting. I will look into it to make sure the push works correctly.

Reply:
why would it not happen? :D

Reply:
imo, formatting code shouldn't be a requirement before pushing code, and I'd rather rely on auto-formatting to make sure that the PRs are formatted properly before they get merged